packages/hash/api/src/auth/index.ts

```diff
@@ -1,19 +1,38 @@
 import { Express } from "express";
-import { DbAdapter } from "../db";
-import { setupPassport } from "./passport";
-import { setupSession, SessionConfig } from "./session";
-import { Config as PgConfig } from "../db/postgres";
+import { AxiosError } from "axios";
+import { Session } from "@ory/client";
+import { kratosSdk } from "./ory-kratos";
+import { User } from "../model";
 
-const setupAuth = (
-  app: Express,
-  sessConfig: SessionConfig,
-  pgConfig: PgConfig,
-  db: DbAdapter,
-) => {
-  // setup session related middleware
-  setupSession(app, sessConfig, pgConfig);
-  // setup passport related middleware and routes
-  setupPassport(app, db);
+declare global {
+  namespace Express {
+    interface Request {
+      session: Session | undefined;
+      user: User | undefined;
+    }
+  }
+}
+
+const setupAuth = (app: Express) => {
+  app.use(async (req, _res, next) => {
+    const kratosSession = await kratosSdk
+      .toSession(undefined, req.header("cookie"))
+      .then(({ data }) => data)
+      .catch((err: AxiosError) => {
+        // 403 on toSession means that we need to request 2FA
+        if (err.response && err.response.status === 403) {
+          /** @todo: figure out if this shoulds be handled here, or in the next.js app (when implementing 2FA) */
+        }
+        return undefined;
+      });
+
+    if (kratosSession) {
+      req.session = kratosSession;
+      /** @todo: attach User model class instance to the `req` object */
```

Comment:
This will be implemented in [this follow-up task](https://app.asana.com/0/1202621375435411/1202693393444197/f), when we are storing users in the graph